### PR TITLE
Updated help with formatting text

### PR DESCRIPTION
**Pull request by kennyevil**
## Changes in this PR

Updated the content of  the "help with formatting" section for non-templated emails.

## Screenshots of UI changes
Screenshot 2022-02-17 at 13-44-42 Buy for your school

### CODE REVIEW

**Comment by kennyevil** [comment]
[Update the 'Help with formatting' information on the Non templated email](https://trello.com/c/blYj16xW/335-update-the-help-with-formatting-information-on-the-non-templated-email)

File: app/views/support/cases/emails/content/_help_with_formatting.en.html.erb [hunk 1]
```diff
@@ -6,49 +6,74 @@
...
-      <p class="govuk-body" style="letter-spacing: 1px;">---</p>
-      <p class="govuk-body">Second paragraph</p>
+      [UK Parliament](http://www.parliament.uk).
     </div>
 
-    <h2 class="heading-medium">Links and URLs</h2>
+    <h2 class="heading-medium">Email addresses</h2>
     <p class="bottom-gutter-1-3">
-      Always use full URLs, starting with https://
+      Email addresses will automatically fromat as a link:
```
Comment (peterdavidhamilton) [inline app/views/support/cases/emails/content/_help_with_formatting.en.html.erb:54]: `format`

File: app/views/support/cases/emails/content/_help_with_formatting.en.html.erb [hunk 2]
```diff
@@ -6,49 +6,74 @@
...
     <div class="govuk-!-margin-bottom-6">
       <div class="govuk-inset-text govuk-!-margin-top-0">
         * point 1<br> * point 2<br> * point 3<br>
       </div>
     </div>
 
+    <h3 class="heading-small">Numbered lists</h3>
     <p class="bottom-gutter-1-3">
-      To add inset text, use a caret:
+      Use a number followed by a full stop to make numbered lists. The line above <b> * point 1 </b> should be blank:
```
Comment (quaypho) [inline app/views/support/cases/emails/content/_help_with_formatting.en.html.erb:36]: The last sentence should be 'The line above <b> 1. Add numbers </b> should be blank:

File: app/views/support/cases/emails/content/_help_with_formatting.en.html.erb [hunk 3]
```diff
@@ -6,49 +6,74 @@
...
     </p>
     <div class="govuk-!-margin-bottom-6">
       <div class="govuk-inset-text govuk-!-margin-top-0">
-        ^ You must tell us if your circumstances change
+        1. Add numbers<br>2. Check numbers<br>3. Love numbers<br>
       </div>
     </div>
 
+    <h2 class="heading-medium">Links and URLs</h2>
```
Comment (quaypho) [inline app/views/support/cases/emails/content/_help_with_formatting.en.html.erb:44]: I think this should be a h3 for consistency

File: app/views/support/cases/emails/content/_help_with_formatting.en.html.erb [hunk 4]
```diff
@@ -6,49 +6,74 @@
...
     </p>
     <div class="govuk-inset-text govuk-!-margin-top-0">
-      <p class="govuk-body">First paragraph</p>
-      <p class="govuk-body" style="letter-spacing: 1px;">---</p>
-      <p class="govuk-body">Second paragraph</p>
+      [UK Parliament](http://www.parliament.uk).
     </div>
 
-    <h2 class="heading-medium">Links and URLs</h2>
+    <h2 class="heading-medium">Email addresses</h2>
```
Comment (quaypho) [inline app/views/support/cases/emails/content/_help_with_formatting.en.html.erb:52]: I think this should be a h3 for consistency

File: app/views/support/cases/emails/content/_help_with_formatting.en.html.erb [hunk 5]
```diff
@@ -6,49 +6,74 @@
...
+    <h3 class="heading-small">Inset text</h3>
+    <p class="bottom-gutter-1-3">
+      To add inset text use a caret. The line above should be blank:
+    </p>
+    <div class="govuk-!-margin-bottom-6">
+      <div class="govuk-inset-text govuk-!-margin-top-0">
+        ^This is useful information that's worth knowing.
+      </div>
+    </div>
+
```
Comment (quaypho) [inline app/views/support/cases/emails/content/_help_with_formatting.en.html.erb:69]: There is a missing section. The 'New line' section should be here.

**Review by quaypho — CHANGES_REQUESTED** [review]
Just a few bits and bobs need editing/updating

File: app/views/support/cases/emails/content/_help_with_formatting.en.html.erb (hunk 5 above)
Comment (kennyevil) [reply app/views/support/cases/emails/content/_help_with_formatting.en.html.erb:69]: I've added that in now.

**Review by peterdavidhamilton — APPROVED** [review]
FYI the schema is getting commit too

File: app/views/support/cases/emails/content/_help_with_formatting.en.html.erb [hunk 6]
```diff
@@ -6,34 +6,68 @@
...
     </p>
     <div class="govuk-!-margin-bottom-6">
       <div class="govuk-inset-text govuk-!-margin-top-0">
         * point 1<br> * point 2<br> * point 3<br>
       </div>
     </div>
 
+    <h3 class="heading-small">Numbered lists</h3>
+    <p class="bottom-gutter-1-3">
+      Use a number followed by a full stop to make numbered lists. The line above <b> * 1. Add numbers </b> should be blank:
```
Comment (quaypho) [inline app/views/support/cases/emails/content/_help_with_formatting.en.html.erb:36]: Please remove the *

File: app/views/support/cases/emails/content/_help_with_formatting.en.html.erb [hunk 7]
```diff
@@ -43,12 +77,9 @@
       <p class="govuk-body">Second paragraph</p>
     </div>
 
-    <h2 class="heading-medium">Links and URLs</h2>
+    <h3 class="heading-small">New lines</h3>
     <p class="bottom-gutter-1-3">
-      Always use full URLs, starting with https://
+      adding two spaces at the end of a line will create a 'soft' new line (equivalent to pressing the return key in a text editor)
```
Comment (quaypho) [inline app/views/support/cases/emails/content/_help_with_formatting.en.html.erb]: Please can we change this to:

To add a new line, use 2 spaces and press **enter/return**

This is the first line (add 2 spaces and press **enter/return)**
This is the second line

**Review by quaypho — CHANGES_REQUESTED** [review]

**Review by shafiqahmedkhan — APPROVED** [review]

**Pull request merged**